scripts/scil_get_subset_streamlines.py

```diff
@@ -2,12 +2,14 @@
 # -*- coding: utf-8 -*-
 import argparse
 
-from nibabel.streamlines import load, save, Tractogram
-import numpy as np
+from dipy.io.streamline import save_tractogram
+from scilpy.io.streamlines import load_tractogram_with_reference
```

Comment:
This `scilpy` import should be with the others (add newline before, remove newline after)